From d5d7fd12a6f3613e3be14238aac93e4a84eade33 Mon Sep 17 00:00:00 2001
From: Wang Panzhenzhuan <randy.wang@rock-chips.com>
Date: Thu, 28 May 2020 16:02:54 +0800
Subject: [PATCH] ARM: dts: rockchip: 3288-android fix CIF_ISP_PIC_SIZE_ERROR
 when use dualview

The defaultly SYS_STATUS_ISP & SYS_STATUS_DUALVIEW ddr rate 396MHz
is low when use dualview and open camera simultaneously;
it causes CIF_ISP_PIC_SIZE_ERROR, so raise it;

Signed-off-by: Wang Panzhenzhuan <randy.wang@rock-chips.com>
Change-Id: I7c919404a31ac8b315b575ed4925e9268fa64eee
---
 arch/arm/boot/dts/rk3288-android.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/rk3288-android.dtsi b/arch/arm/boot/dts/rk3288-android.dtsi
index 2244e7d09665..98f642c306b4 100644
--- a/arch/arm/boot/dts/rk3288-android.dtsi
+++ b/arch/arm/boot/dts/rk3288-android.dtsi
@@ -89,8 +89,8 @@
 			SYS_STATUS_VIDEO_4K_10B	528000
 			SYS_STATUS_PERFORMANCE	528000
 			SYS_STATUS_BOOST	396000
-			SYS_STATUS_DUALVIEW	396000
-			SYS_STATUS_ISP		396000
+			SYS_STATUS_DUALVIEW	528000
+			SYS_STATUS_ISP		528000
 		>;
 		auto-min-freq = <396000>;
 		auto-freq-en = <0>;
-- 
2.35.3

